kiwipy/rmq/defaults.py

```diff
@@ -18,4 +18,4 @@
 TASK_PREFETCH_COUNT = 0
 
 ENCODER = partial(yaml.dump, encoding='utf-8')
-DECODER = yaml.load
+DECODER = lambda _: yaml.load(_, Loader=yaml.Loader)
```

Comment:
Since the value of ``_`` is actually used, maybe we should give it a name? Keeping with the naming from ``pyyaml``, it would be ``lambda stream: yaml.load(stream, Loader=yaml.Loader)``.

Reply:
Or better yet, same as above: ``partial(yaml.load, Loader=yaml.Loader)``. Totally bike-shedding here though, feel free to merge whenever.

Reply:
Haha, yes, talk about not looking at the line above!